apps/condo/domains/common/components/containers/FormList.tsx

```diff
@@ -429,18 +432,21 @@ const BaseModalForm: React.FC<IBaseModalFormProps> = ({
         centered
         {...modalProps}
     >
-        <FormWithAction {...props}>
-            {
-                ({ handleSave, form }) => {
-                    handleSaveRef.current = handleSave
-                    return (
-                        <>
-                            {typeof children === 'function' ? children(form) : children}
-                        </>
-                    )
+        <Space direction={'vertical'} size={40}>
+            {modalNotification}
+            <FormWithAction {...props}>
+                {
+                    ({ handleSave, form }) => {
+                        handleSaveRef.current = handleSave
```

Comment:
Looks like hack

Reply:
Maybe it is...But it's not mine....I just added `Space` here

Reply:
😄